### DISCUSSION TITLE
help, the failed of SBAS-Insar

### DISCUSSION THREAD

**Discussion by pupil-insar** [ASFHyP3/hyp3-docs#470] [Q&A]
The asf website created 189 pairs when I chose the SBAS, and then I submitted them to processing. The results shows that 13 pairs from pending turns to failed. I want to know the failed reason. And I want to konw if I re-submit the failed one and then they become success. wheather the  interference result of them available? wheather they increase the error?

**Comment by jhkennedy** [comment]
@pupil-insar, all failed jobs should have logs that may give you insight into the failures. If you can let me know what your Earthdata Login username is, I can have a look.
